duckdns/rootfs/root/hooks.sh

```diff
@@ -14,6 +14,9 @@ deploy_challenge() {
     local DOMAIN="${1}" TOKEN_FILENAME="${2}" TOKEN_VALUE="${3}" ALIAS
     ALIAS="$(jq --raw-output --exit-status "[.aliases[]|{(.domain):.alias}]|add.\"$DOMAIN\"" $CONFIG_PATH)" || ALIAS="$DOMAIN"
 
+	echo "Processing domain: $DOMAIN"
+	echo ""
```

Comment:
We typically use `bashio::log`, e.g.

```suggestion
	bashio::log.info "Processing domain: $DOMAIN"
```